test/first-order/no-arguments.coffee

```diff
@@ -64,12 +64,23 @@ module.exports = async ->
 
   run "sass"
 
+  define "handlebars", async ->
+    yield go [
+      glob "*.hb", src
```

Comment:
This may want to be a regex array that may search for `[.hb, .hbs, .handlebars]`. I have seen projects use one of each of those extensions.